CMakeLists.txt

```
@@ -170,6 +171,10 @@ add_subdirectory(config)
 
 add_subdirectory(doc)
 
+if(BUILD_APPLICATIONS)
```

If previous suggestion is applied:
```suggestion
if(EVEREST_BUILD_APPLICATIONS)
```